.ci/azure-pipelines-daily.yml

```diff
@@ -0,0 +1,41 @@
+schedules:
+  - cron: '0 10 * * *'
+    displayName: Daily late night PST (GMT-8)
+    branches:
+      include:
+        - helics3
+
+jobs:
+  - job: Daily Linux
+    strategy:
+      matrix:
+        all-tests:
+          containerImage: 'helics/buildenv:ubuntu18.04-default-builder'
```

Comment:
what compiler does the default builder use?

Reply:
gcc7 - this is the build using the default version of all Ubuntu 18.04 packages.